### PR TITLE
configure conda installation

### PR DESCRIPTION
**Pull request by josephvitti**

### CODE REVIEW

File: requirements-docs.txt [hunk 1]
```diff
@@ -1,10 +0,0 @@
-biopython==1.65
```
Comment (tomkinsc) [inline requirements-docs.txt:1]: We'll still need `requirements-docs.txt`, since ReadTheDocs relies on it (for now...).

Comment (josephvitti) [inline requirements-docs.txt:1]: AH yes. That makes sense. I'll amend!

**Comment by josephvitti** [comment]
hmm, it looks like Travis wants there to be a requirements.txt when it runs travis/install-pip.sh, causing these checks to fail. @tomkinsc I can easily just restore the file (and ensure consistency between it and conda-enrionment.yml), but wanted to check if this should be the case -- I haven't used Travis before, but am guessing we just need to tell it not to expect requirements.txt.

**Comment by tomkinsc** [comment]
Oh, that's right. I held off on updating Travis to use a conda environment because of a bug conda had with importing its own environment files. Let me see if I can condafy Travis on a separate branch. If not we'll have to keep requirements.txt for a little while longer.

File: conda-environment.yml [hunk 2]
```diff
@@ -58,6 +58,7 @@ dependencies:
   - flake8==2.6.0
   - mccabe==0.5.0
   - mock==2.0.0
+  - numpy==1.11.0
```
Comment (tomkinsc) [inline conda-environment.yml:61]: numpy [is included](https://github.com/broadinstitute/cms/blob/master/conda-environment.yml#L32) in the conda package list. Do we need to also include it in the pip dependencies?

**Comment by tomkinsc** [comment]
Ok, Travis is now configured to use the `conda-environment.txt` file, and I've merged the branch with that change into master, and updated this branch to master.

File: conda-environment.yml (hunk 2 above)
Comment (josephvitti) [inline conda-environment.yml:61]: Hmm, I've been having some import errors that I haven't been able to get to the bottom of. I've tried out a whole slew of variations on the installation procedure, and it appears I always end up either unable to import: pysam, boltons.timeutils, playhouse.fields, or numpy. This latter module appears to be the recurrent issue when I follow what I understand to be the correct installation procedure. Looking at conda-environment.yml, I noticed that pysam, boltons and playhouse are all listed in the conda list as well as the pip dependencies, so I thought I'd add numpy in there and see if it makes a difference. Hopefully I'll be able to get to the bottom of this today.

**Pull request merged**